core/src/main/java/jenkins/tasks/SimpleBuildWrapper.java

```diff
@@ -120,13 +147,14 @@ public void setDisposer(@NonNull Disposer disposer) {
         /**
          * Attempt to clean up anything that was done in the initial setup.
          * @param build a build being run
-         * @param workspace a workspace of the build
-         * @param launcher a way to start commands
+         * @param workspace a workspace of the build; may be {@code null} if is allowed by the corresponding wrapper
+         * @param launcher a way to start commands; may be {@code null} if is allowed by the corresponding wrapper
```

Comment:
```suggestion
         * @param workspace a workspace of the build; may be {@code null} if allowed by the corresponding wrapper
         * @param launcher a way to start commands; may be {@code null} if allowed by the corresponding wrapper
```

Reply:
Yes, I seem to have either lost an `it` or failed to remove the `is`.